README.md

```diff
@@ -26,6 +27,26 @@ the function and run automatically when you commit through a GitHub action.
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
 
+This implementation of Depth First Seach has complexity $\Theta (V^2)$.
+There is a while loop that iterates over every V in the worst case. Inside the
+while loop there is a for loop that in the worst case is iterating over a node with
+each other node connected to it, resulting in a runtime of V as well. There is also 
+2 for loops outside the while loop calculating the actual path to the node but they 
+have the same complexity as the while loop so we can ignore them. Therefore we can 
+analyze this using loop logic (Ex: a loop inside a loop has runtime n^2) resulting 
+in runtime of $\Theta (V^2)$.
```

Comment:
Would any of this depend on the number of edges?

Reply:
Yeah, you're right, I realized that the for loop is actually iterating over edges. The readme has been updated accordingly